src/black/nodes.py

```diff
@@ -778,12 +779,21 @@ def is_import(leaf: Leaf) -> bool:
     )
 
 
+is_type_comment_regex = re.compile(r'^#[\s\t]+type[\s\t]*:')
+
+
 def is_type_comment(leaf: Leaf, suffix: str = "") -> bool:
     """Return True if the given leaf is a special comment.
     Only returns true for type comments for now."""
     t = leaf.type
     v = leaf.value
-    return t in {token.COMMENT, STANDALONE_COMMENT} and v.startswith("# type:" + suffix)
+    if t not in {token.COMMENT, STANDALONE_COMMENT}:
+        return False
+
+    if not v.startswith("#"):
+        return False
```

Comment:
can combine if statements of ln790 and ln793 into one if statement since they both return the same thing

Reply:
You're right! I started thinking about guardian clauses and got carried away :) 
Fixed!